### PR TITLE
Add documentation of Skygear Chat UIKit

### PR DESCRIPTION
**Pull request by ben181231**
connect SkygearIO/chat#77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,11 @@ it, simply add the following line to your Podfile:
 pod "SKYKitChat"
 ```
 
+## Documentation
+
+- Core (Coming soon...)
+- [UIKit](https://github.com/kygearIO/chat-SDK-iOS/blob/master/docs/UIKit/index.md)
```
Comment (royuen) [inline README.md]: wrong URL is given

kygearIO should be skygearIO

Comment (ben181231) [reply README.md]: Oh...

**Comment by ben181231** [comment]
@royuen Updated.

**Comment by cheungpat** [comment]
@ben181231 still expecting my feedback on this?

**Comment by ben181231** [comment]
@cheungpat Yes please. 😅

File: docs/UIKit/ConversationListView.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+extension ConversationListDemoViewController: SKYChatConversationListViewControllerDataSource {
+    func listViewController(_ controller: SKYChatConversationListViewController,
+                            avatarImageForConversation conversation: SKYConversation,
+                            atIndexPath indexPath: IndexPath) -> UIImage?
+    {
+        // implement a store for storing conversation data
+        var image = AvatarStore.shared.avatar(forConversation: conversation)
+        if image == nil {
+            image = /* fetch the image */
```
Comment (cheungpat) [inline docs/UIKit/ConversationListView.md]: this example is not very good when it comes to fetching and updating async image, plus i prefer example to compile cleanly, which this snippet is clearly not.

Comment (ben181231) [reply docs/UIKit/ConversationListView.md]: So just assume that all the avatars are already in `AvatarStore` ?

**Comment by ben181231** [comment]
Updated.

**Pull request merged**